contrib/win32/win32compat/fileio.c

```diff
@@ -638,22 +639,53 @@ int
 fileio_stat(const char *path, struct _stat64 *buf)
 {
 	wchar_t* wpath = NULL;
-	int r = -1;
-
-	if ((wpath = utf8_to_utf16(path)) == NULL)
-		fatal("failed to covert input arguments");
-
-	r = _wstat64(wpath, buf);
-
-	/*
-	* If we doesn't have sufficient permissions then _wstat64() is returning "file not found"
```

Comment:
I would suggest to remove all this code and have _wstat64() but before making that attempt call GetFileAttributesExW() since we are just worried about the wrong error message.. we shouldn't be re-writing _wstat64() implementation again? If GetFileAttributesExW() success then call _wstat64